recipes/r-rdimtools/meta.yaml

```
@@ -0,0 +1,121 @@
+{% set version = '1.1.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rdimtools
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rdimtools_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rdimtools/Rdimtools_{{ version }}.tar.gz
+  sha256: 867630ee2588ce22e22614b66c5b60fb0f7e8b74a08c25c3ed66c9e4bd73882e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/r.dll'  # [win]
+    - '*/Rlapack.dll'  # [win]
+    - '*/Rblas.dll'    # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}            # [not win]
+    - {{ compiler('m2w64_c') }}      # [win]
+    - {{ compiler('cxx') }}          # [not win]
+    - {{ compiler('m2w64_cxx') }}    # [win]
+    - {{ posix }}filesystem          # [win]
+    - {{ posix }}make
+    - {{ posix }}sed                 # [win]
+    - {{ posix }}coreutils           # [win]
+    - {{ posix }}zip                 # [win]
+    - xorg-libxfixes                 # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cross-r-base {{ r_base }}      # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-admm
+    - r-cvxr >=1.0
+    - r-mass
+    - r-rann
+    - r-rspectra
+    - r-rcpp >=0.12.15
+    - r-rcpparmadillo
+    - r-rcppde
+    - r-rcppdist
+    - r-rdpack
+    - r-maotai >=0.2.4
+    - r-mclustcomp
+    - libblas
+    - liblapack
+    - llvm-openmp
+    - m2w64-libwinpthread-git  # [win]
+    - r-rgl
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-admm
+    - r-cvxr >=1.0
+    - r-mass
+    - r-rann
+    - r-rspectra
+    - r-rcpp >=0.12.15
+    - r-rcpparmadillo
+    - r-rcppde
+    - r-rcppdist
+    - r-rdpack
+    - r-maotai >=0.2.4
+    - r-mclustcomp
+    - m2w64-libwinpthread-git  # [win]
```

FYI, in the future, this should be expressed as:

```suggestion
    - {{ native }}libwinpthread-git  # [win]
```